integration/kubernetes/k8s-memory.bats

```diff
@@ -6,8 +6,12 @@
 #
 
 load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+TEST_INITRD="${TEST_INITRD:-no}"
 
 setup() {
+	issue="https://github.com/kata-containers/runtime/issues/1127"
```

Comment:
as the issue is the same for all tests here, it could go outside the `setup` to not have it on each of the tests. Just like the `TEST_INITRD` variable. Same for all other .bats files below.